.github/workflows/build.yml

```diff
@@ -24,7 +24,7 @@ env:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Can you explain why this is needed? It sounds like a good idea to use a fixed version regardless, but it also sounds like it should be a separate PR altogether.

Reply:
ubuntu-latest (18.04) only has SWIG 3.0, and has trouble parsing static_asserts: https://github.com/nwnxee/unified/runs/1677788148#step:10:83.

20.04 fixes this as it has access to SWIG 4.